StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetConfiguration.swift

```diff
@@ -193,6 +193,8 @@ extension PaymentSheet {
         // MARK: Internal
         internal var linkPaymentMethodsOnly: Bool = false
 
+        @_spi(STP) public var forceEnableNativeLink: Bool = false
```

Comment:
nit: is "force" really necessary?   Maybe `nativeLinkEnabled`.. but if we do want the word 'force', then maybe `forceNativeLinkEnabled`

Reply:
This is an override switch for testing, it's going away once the server-side behavior is implemented.